includes/class-widget.php

```diff
@@ -41,6 +41,8 @@ public function widget( $args, $instance ) {
 
 		global $post;
 
+		$license_key = get_option( 'republication_tracker_tool_license' );
```

Comment:
Because the `republication_tracker_tool_license` is a new option, sites that use the widget already and don't have the option set yet probably need a default value for this to avoid this error further down:
`Warning: Undefined array key 0 in republication-tracker-tool/includes/class-widget.php on line 75`.

Reply:
@naxoc thanks for your thoughtful review! I've added `cc-by-nd` as the default, since it was the only license available before.